mdpow/tests/test_equilibration.py

```diff
@@ -0,0 +1,57 @@
+import mdpow.equil
+import tempfile
+import os
+import shutil
+import mdpow.run
+import gromacs.environment
+gromacs.environment.flags['capture_output'] = True
+
+class TestEquilibration(object):
+    sim_filename = ".simulation"
+    sims = {"water": mdpow.equil.WaterSimulation,  
+            "octanol":mdpow.equil.OctanolSimulation, 
+            "cyclohexane":mdpow.equil.CyclohexaneSimulation}
+    
+    def setup(self):
```

Comment:
I have found that it works when I run it.
On May 28, 2016 10:21 PM, "Oliver Beckstein" <notifications@github.com>
wrote:

> In mdpow/tests/test_equilibration.py
> <https://github.com/Becksteinlab/MDPOW/pull/39#discussion_r64997763>:
>
> > @@ -0,0 +1,57 @@
> > +import mdpow.equil
> > +import tempfile
> > +import os
> > +import shutil
> > +import mdpow.run
> > +import gromacs.environment
> > +gromacs.environment.flags['capture_output'] = True
> > +
> > +class TestEquilibration(object):
> > +    sim_filename = ".simulation"
> > +    sims = {"water": mdpow.equil.WaterSimulation,
> > +            "octanol":mdpow.equil.OctanolSimulation,
> > +            "cyclohexane":mdpow.equil.CyclohexaneSimulation}
> > +
> > +    def setup(self):
>
> Is setup() actually executed by *py.test*? According to classic
> xunit-style setup <https://pytest.org/latest/xunit_setup.html> such
> methods have to be called differently (maybe setup_class()?) *or* the
> class must be derived from unitest.TestCase.
>
> Apprently, the recommended way to do fixtures is py.test fixtures
> <https://pytest.org/latest/fixture.html#fixture> ... but I find it rather
> convoluted.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Becksteinlab/MDPOW/pull/39/files/0d0332fc500df10615e5324177b73f28125037a0#r64997763>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGwcbJzLJHf_KVkrU_pYU3Qqq0FZWUDzks5qGSJVgaJpZM4Ik-aG>
> .
>